introduction.md

```diff
@@ -1,3 +1,7 @@
 Hey there! I'm Frances. :)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?